src/main/java/org/team1540/elmo/drivetrain/DriveTrain.java

```diff
@@ -0,0 +1,37 @@
+package org.team1540.elmo.drivetrain;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.team1540.elmo.Constants;
+
+public class DriveTrain extends SubsystemBase {
+
+    TalonFX motorRightFront = new TalonFX(Constants.RIGHT_FRONT_MOTOR_CAN_ID);
+    TalonFX motorRightBack = new TalonFX(Constants.RIGHT_BACK_MOTOR_CAN_ID);
+    TalonFX motorLeftFront = new TalonFX(Constants.LEFT_FRONT_MOTOR_CAN_ID);
+    TalonFX motorLeftBack = new TalonFX(Constants.LEFT_BACK_MOTOR_CAN_ID);
```

Comment:
I don't think that you will have Falcons on this robot. I believe you'll be using Victor SPX speed controllers for brushed motors, make sure to confirm with Controls/Fab though. The SPXes will still be part of the Phoenix library, so the interface should be pretty similar to the Talons